apps/flux-system/aat/base/kustomization.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
 - ../../base
-- gotk-components.yaml
+- ../../base/gotk-components.yaml
 - aks-sops-aadpodidentity.yaml
```

Comment:
Could you test with this line removed on one of them? I feel like this line shouldn't be necessary because of `../../base`